gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Need to fix Copyright. You should set yourself up with `repoman` or `pkgdev`/`pkgcheck` to have these things fixed automatically for you plus get initial QA checks.

Reply:
Thank you for the thorough review. To be honest I never really used these tools up to now. But when trying a `repoman -dx full` it just complains about the deprecated dependency:
```
$ repoman -dx full

RepoMan scours the neighborhood...
  dependency.deprecated         6
   gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild: '>=gnome-base/gnome-vfs-2.0.0'
   gnome-extra/gnome-commander/gnome-commander-1.12.1.ebuild: '>=gnome-base/gnome-vfs-2.0.0[samba]'
```
Is this expected? The other issues which you mentioned like the date in line 1, the ordering and all that were not mentioned by repoman.

Reply:
Yes, of those tools `pkgcheck` would be the most thorough; `repoman fix` would fix basic QA issues and `repoman commit` prepares half of a git workflow compliant git summary for you to complete. Both don't mind style best practises that we communicate as part of our review.